data/grc/Erbse_1961_Aischylos-Persai,HeptaepiThebas,Hiketides,Prometheusdesmotes,Agamemnon,Choephoroi,Eumenides/stemma.gv

```diff
@@ -0,0 +1,27 @@
+digraph {
+archetyp[color="grey", label="ω"] 
+phi [label="Φ"] 
+pi[color="grey", label="π"] 
+beta [color="grey", label="β"] 
+xi [color="grey", label="ξ"] 
+rho [color="grey", label="ρ"] 
+sigma [color="grey", label="σ"] 
+psi [color="grey", label="ψ"] 
```

Comment:
- Is phi really extant ? 
- Why do we have lost nodes without descent (pi, beta, etc.) ?

Reply:
As you mention in another comment, they end the tree at some hypothetical archetype.